tests/unit/s2n_stream_cipher_null_test.c

```diff
@@ -22,6 +22,7 @@ int main(int argc, char **argv)
 {
     BEGIN_TEST();
     EXPECT_SUCCESS(s2n_disable_tls13_in_test());
+    struct s2n_session_key session_key = { 0 };
```

Comment:
Sharing potentially mutable resources between tests can be dangerous. Tests should be isolated from one another. Each test should have its own session_key.

Reply:
fixed at https://github.com/aws/s2n-tls/pull/5268/commits/135ecd2a1543507155f8b6efe8ed5cfb42513a84